docs/examples/800_istio.adoc

```diff
@@ -0,0 +1,183 @@
+= Istio Support
+
+== Overview
+
+Coherence Operator 3.1.15 and later works with Istio 1.9.1 and later. You can run the operator and Coherence cluster managed by the operator with Istio sidecar injection.  Coherence caches can be accessed from outside the Coherence cluster via Coherence*Extend, REST, and other supported Coherence clients.  
+
+NOTE: 
+----
```

Comment:
Delete the `----` line and change `Note:` to `== Prerequisites`

Reply:
changed